conf/modules.config

```diff
@@ -162,6 +164,115 @@ process {
         publishDir = [ enabled: false ]
     }
 
+    withName: '.*TAXID_BAM_FASTA_SHORTREAD:SAMTOOLS_IDXSTATS' {
+        publishDir = [ enabled : false ]
+    }
+
+    withName: '.*TAXID_BAM_FASTA_SHORTREAD:SUBSET_BAM_PASS' {
+        ext.prefix = { "${meta.id}_${meta.taxid}"}
+        ext.args = '-bh'
+        publishDir = [
+            path: { "$params.outdir/pathogens/taxid_bam/" },
+            enabled : false
+        ]
+    }
+
+    withName: '.*TAXID_BAM_FASTA_SHORTREAD:SAMTOOLS_SORT_PASS' {
```

Comment:
What does PASS refer to here?

Reply:
It refers to the the number of reads mapped to a pathogen genome that is greater than `params.min_read_counts`.  I commented on it in[ line 73](https://github.com/genomic-medicine-sweden/meta-val/blob/03b5e79e887e030b65542d787cfe2ea6a7444c2b/subworkflows/local/taxid_bam_fasta.nf#L73) and [line 95](https://github.com/genomic-medicine-sweden/meta-val/blob/03b5e79e887e030b65542d787cfe2ea6a7444c2b/subworkflows/local/taxid_bam_fasta.nf#L95)